src/avio.jl

```diff
@@ -668,38 +663,17 @@ if have_avdevice()
 end
 
 try
-    if isa(Main.Images, Module)
-        # Define read and retrieve for Images
-        global retrieve, retrieve!, read!, read
-        for r in [:read, :retrieve]
-            r! = Symbol("$(r)!")
-
-            @eval begin
-                # read!, retrieve!
-                $r!(c::VideoReader, img::Main.Images.Image) = ($r!(c, Main.Images.data(img)); img)
-
-                # read, retrieve
-                function $r(c::VideoReader, ::Type{Main.Images.Image}, colorspace="RGB", colordim=1, spatialorder=["x","y"])
-                    img = Main.Images.colorim($r(c::VideoReader))
-                end
-            end
-        end
-    end
-end
-
-try
     if isa(Main.ImageView, Module)
```

Comment:
I'd rather just remove this and put it in a separate package (although that doesn't have to happen in this PR).